geomesa-arrow/geomesa-arrow-dist/src/main/assemblies/component.xml

```diff
@@ -22,7 +22,6 @@
             <useTransitiveDependencies>false</useTransitiveDependencies>
             <useStrictFiltering>true</useStrictFiltering>
             <includes>
-                <include>org.locationtech.geomesa:geomesa-arrow-gs-plugin*</include>
                 <include>org.locationtech.geomesa:geomesa-process-wps*</include>
```

Comment:
If these blocks are all standardized now, can the configuration be factored up?

Reply:
possibly... it's theoretically possibly to share common 'component' descriptors and import them in different places. could be used for e.g. readmes, etc that we include lower down as well. we can add a ticket to do that, i'd rather not do it as part of this pr though.